DEVELOPER_GUIDE.md

```diff
@@ -67,6 +67,10 @@ If it's the only version of node installed, it will automatically be set to the
 $ npm i -g corepack
 ```
 
+```bash
+$ corepack prepare yarn@<version> --activate
```

Comment:
let's just combine them in a single code block.

```suggestion
$ corepack prepare yarn@<version> --activate
```

Reply:
Don't we also need to include how to get the version?

```
Take a look at the [latest Yarn release](https://github.com/yarnpkg/berry/releases/latest), note the version number, and run:
```

Reply:
Thank you! Great suggestions, working on them.